laplace/lllaplace.py

```diff
@@ -117,7 +117,7 @@ def fit(self, train_loader, override=True):
         self.mean = parameters_to_vector(self.model.last_layer.parameters()).detach()
 
     def _glm_predictive_distribution(self, X):
```

Comment:
This method can be removed with a minor refactor in some places, but I'll just do that in a later PR.

Reply:
Ah ok, cool!